Session1.Rmd

```diff
@@ -217,7 +222,7 @@ You can create a vector with a `c()` function.
 numeric_vector <- c(2, 6, 3) # vector of numbers - numeric data type.
 numeric_vector
 
-character_vector <- c('banana', 'apple', 'orange') # vector of words - more precisely strings of characters- character data type
+character_vector <- c('banana', 'apple', 'orange') # vector of words - or strings of characters- character data type
```

Comment:
Depending on who participates, it might be a good idea to briefly explain the difference between an integer or float/double, and state that sometimes a character is called a string. It is a minor thing, but it might resolve confusion in some instances (e.g. when you explain factors below -line 288- you also talk about integers.

Reply:
That's a good idea, but I think I removed the integer all together in interest of time. I adapted the line where I write about factors so that it says 'numbers' instead of integer vector. 

I'll try to remember to mention that 'strings of characters' are often just called 'strings'.